backend/src/apiserver/server/experiment_server_test.go

```diff
@@ -124,30 +141,23 @@ func TestCreateExperimentV1_Unauthorized(t *testing.T) {
 	defer clients.Close()
 
 	server := ExperimentServer{resourceManager: resourceManager, options: &ExperimentServerOptions{CollectMetrics: false}}
-	experiment := &apiV1beta1.Experiment{
+	experiment := &apiv1beta1.Experiment{
 		Name:        "exp1",
 		Description: "first experiment",
-		ResourceReferences: []*apiV1beta1.ResourceReference{
+		ResourceReferences: []*apiv1beta1.ResourceReference{
 			{
-				Key:          &apiV1beta1.ResourceKey{Type: apiV1beta1.ResourceType_NAMESPACE, Id: "ns1"},
-				Relationship: apiV1beta1.Relationship_OWNER,
+				Key:          &apiv1beta1.ResourceKey{Type: apiv1beta1.ResourceType_NAMESPACE, Id: "ns1"},
+				Relationship: apiv1beta1.Relationship_OWNER,
 			},
-		}}
+		},
+	}
 
-	_, err := server.CreateExperimentV1(ctx, &apiV1beta1.CreateExperimentRequest{Experiment: experiment})
+	_, err := server.CreateExperimentV1(ctx, &apiv1beta1.CreateExperimentRequest{Experiment: experiment})
 	assert.NotNil(t, err)
-	resourceAttributes := &authorizationv1.ResourceAttributes{
-		Namespace: "ns1",
-		Verb:      common.RbacResourceVerbCreate,
-		Group:     common.RbacPipelinesGroup,
-		Version:   common.RbacPipelinesVersion,
-		Resource:  common.RbacResourceTypeExperiments,
-		Name:      experiment.Name,
-	}
-	assert.EqualError(
+	assert.Contains(
 		t,
-		err,
-		wrapFailedAuthzRequestError(wrapFailedAuthzApiResourcesError(getPermissionDeniedError(userIdentity, resourceAttributes))).Error(),
+		err.Error(),
+		"PermissionDenied: User 'user@google.com' is not authorized with reason: this is not allowed (request: &ResourceAttributes{Namespace:ns1,Verb:create,Group:pipelines.kubeflow.org,Version:v1beta1,Resource:experiments,Subresource:,Name:exp1,})",
```

Comment:
nit: I think the old way of defining `resourceAttributes` outside the assert is more readable.

Reply:
>This is about creating source, are we no longer rejecting namespace input in single user mode? I think we probably should. Even if we don't filter on retrieving, we shouldn't allow storing arbitrary value that may or may not be valid.

In a single-user mode we simply replace the namespace with the default value (empty string). New namespaces will not be created. When creating k8s resources, we will use the `POD_NAMESPACE` from the config (Set to `kubeflow`). This logic will probably be more robust especially for some edge cases with people migrating from v1 to v2. No strong opinion here.

Reply:
>nit: I think the old way of defining resourceAttributes outside the assert is more readable.

I might be wrong on this, but IMHO presenting it as a string gives more information to a developer of what error message is expected. No strong opinion here.